UI/BindingSources/XAML/Binding/MainPage.xaml

```diff
@@ -0,0 +1,46 @@
+﻿<Page x:Class="Binding.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Binding"
+	  xmlns:um="using:Uno.Material"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  x:Name="page"
+	  Background="{ThemeResource BackgroundBrush}">
+
+	<Page.DataContext>
+		<local:MainViewModel />
+	</Page.DataContext>
+
+	<StackPanel HorizontalAlignment="Center"
+				VerticalAlignment="Center"
+				utu:SafeArea.Insets="VisibleBounds">
+		<ListView ItemsSource="{Binding Items}">
+			<ListView.ItemTemplate>
+				<DataTemplate>
+					<StackPanel Orientation="Horizontal" Spacing="8">
+						<TextBlock VerticalAlignment="Center" Text="{Binding Text}" />
+						<Button Command="{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}"
+								CommandParameter="{Binding}"
+								Content="Delete" />
+
+						<!--  RelativeSource as TemplatedParent works on Windows but not on Uno  -->
+						<!--  RelativeSource as Self works on Uno but not on Windows  -->
+						<Button Command="{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}"
+								CommandParameter="{Binding DataContext, RelativeSource={RelativeSource Mode=Self}}"
+								Content="Delete" />
+						<StackPanel.ContextFlyout>
+							<MenuFlyout>
+								<MenuFlyout.Items>
+									<!--  The Command bind doesn't work on Uno  -->
+									<MenuFlyoutItem Command="{Binding DataContext.RemoveItemCommand, ElementName=page}"
```

Comment:
Binding the command like this doesn't work on Uno. Any ideas?

@agneszitte @dansiegel @kazo0 @Xiaoy312

Reply:
@eriklimakc  It is normal, it is because your ElementName=page is outside the `DataTemplate` where you are trying to bind the value here. That is why we created the [AncestorBinding & ItemsControlBinding](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/helpers/ancestor-itemscontrol-binding.html) in Uno.Toolkit.

Reply:
Huum, that's strange.

`Command="{Binding DataContext.RemoveItemCommand, ElementName=page}"` works on Windows.

And as mentioned here https://github.com/unoplatform/Uno.Samples/pull/681#discussion_r1595646405 using [AncestorBinding & ItemsControlBinding](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/helpers/ancestor-itemscontrol-binding.html) as `{utu:ItemsControlBinding Path=DataContext.RemoveItemCommand}` or `{utu:AncestorBinding AncestorType=ItemsControl, Path=DataContext.RemoveItemCommand}` doesn't work.

Maybe because it's inside a `MenuFlyout` that is inside a `DataTemplate`?

Reply:
yea... flyout content is in a separate visual tree branch which is why ancestor/itemscontrol-binding wouldnt work here.

Reply:
@Xiaoy312 is there another way of making that Binding work for that case?

Reply:
@jeromelaban, @MartinZikmund 
If `Command="{Binding DataContext.RemoveItemCommand, ElementName=page}"` works on Windows inside the MenuFlyout than we would need to match that behavior on Uno right?

Related code:
https://github.com/unoplatform/Uno.Samples/blob/efdbeebbd7693c98cd8e2f5700814668ef6d89f5/UI/BindingSources/XAML/Binding/MainPage.xaml#L34

Reply:
It should indeed, yes.

Reply:
@eriklimakc If you can open a small repro and open an issue in Uno for this one please

Reply:
The issue -> https://github.com/unoplatform/uno/issues/16743
@agneszitte @jeromelaban

Reply:
Thank you a lot for the issue @eriklimakc!